app/assets/stylesheets/_search_result.scss

```diff
@@ -38,22 +39,28 @@
           margin: 10px auto;
           img { margin: 0; }
         }
-      }
+       }
+     }
+    .button-action {
+      float: left;
```

Comment:
Is it possible to use a display:inline-block instead of a float left here? This class only looks to be set on the wording for each button? Or is there another float that set somewhere else that this is trying to override?